bundle/android/strings.xml

```diff
@@ -33,7 +33,6 @@
     <string name="gutenberg_native_add_url" tools:ignore="UnusedResources">Add URL</string>
     <string name="gutenberg_native_add_video" tools:ignore="UnusedResources">Add video</string>
     <string name="gutenberg_native_address_settings" tools:ignore="UnusedResources">Address Settings</string>
-    <string name="gutenberg_native_alt_text" tools:ignore="UnusedResources">Alt Text</string>
```

Comment:
This was removed because it was added in core Gutenberg in https://github.com/WordPress/gutenberg/pull/57909/files#diff-9c023b571e7096ee646e148e20f4bb1454bdfcf0aceefc2a2b613cd604ebca65R34